packages/utils/src/base64.ts

```diff
@@ -1,3 +1,5 @@
+import {Buffer} from "node:buffer";
```

Comment:
Doesn't this break if you load the file in a browser env?

Reply:
I would expect that this is bundler specific, now to be on the safe side probably better to use `import {Buffer} from "node:buffer";` for libs that can end up being used in browsers.
Note that with current config those changes break `browser tests`.

Reply:
One potential solution for this:

* for code running under node, use import
* for code that might be used by other runtimes
  * no import
  * create helper function similar to [uint8arrays](https://github.com/achingbrain/uint8arrays/blob/main/src/util/as-uint8array.ts#L5) abstracting `Buffer` usage (kind of a generalisation of what you did here)